buildwin.ps1 script error building the x64 version of Poco

KevDi opened the pull request:
If build x64 the scripts looks for the _x64.vcproj Files which no longer exists. By removing this the Build is possible.
If only the Build Tools for Visual Studio are available the script does not run because those are not recognized by the Select-VSSetupInstance Script.
Adding the parameter `-product *` also recognizes the Build Tools.

#3740

aleks-f commented on buildwin.ps1:
```diff
@@ -505,8 +505,6 @@ function Build
 	else										 { $extension = 'vcxproj' }
 
 	$platformName = ''
```
It seems that `$platformName` can be completely removed?

KevDi replied:
mmh yes i think so i will remove it, test it and then push the changes

aleks-f commented:
@KevDi can you also try to add a CI entry for this?
It would be another entry like this, just using the `buildwin.ps1` script instead of cmake:
https://github.com/pocoproject/poco/blob/be19dc4a2f30eb97cc9bdd7551460db11cc27353/.github/workflows/ci.yml#L150-L160

KevDi commented:
Ok i will try to add the entry

KevDi commented:
@aleks-f  i tried to add the CI Section but it seems on the Actions Job Run the `-omit` does not work as expected. If i run the same command on my local Windows System it works. But on the Github Actions it still tries to build the Crypto Library.

KevDi commented:
@aleks-f  i test it again with the `Encodings` Project and it seems like on the github actions the check does not work. On my local system the Projects are skipped. But in the github actions no project is skipped and i don't know why

KevDi commented:
@aleks-f  ok i solved that issue. It seems like in the github actions the `split` method should only take exactly one character. I changed it from `.split(",;")`  to `.split(",")` and now the skipping of Projects works.
Should we keep the `,` or the `;` as the seperator for the omit list? I would go with the `;`because it is already mentioned in the help section.

aleks-f commented:
> @aleks-f ok i solved that issue. It seems like in the github actions the `split` method should only take exactly one character. I changed it from `.split(",;")` to `.split(",")` and now the skipping of Projects works. Should we keep the `,` or the `;` as the seperator for the omit list? I would go with the `;`because it is already mentioned in the help section.

I'd go with `,` because it is more intuitive and in line with `make` build. just change the doc, no big deal

aleks-f commented:
now you have some joy with VS2022. try to comment out other CI entries so that a push triggers run only of the entries you are working on, not the whole world
